pkg/local_workflows/auth_workflow.go

```diff
@@ -111,7 +120,36 @@ func entryPointDI(config configuration.Configuration, logger *zerolog.Logger, en
 		}
 
 		fmt.Println(auth.AUTHENTICATED_MESSAGE)
+	} else if strings.EqualFold(authType, auth.AUTH_TYPE_PAT) { // PAT flow
+		instrumentation.AddExtension(authTypeParameter, auth.AUTH_TYPE_PAT)
+
+		pat := config.GetString(ConfigurationNewAuthenticationToken)
+
+		logger.Printf("Unset oauth key %q from config", auth.CONFIG_KEY_OAUTH_TOKEN)
+		config.Unset(auth.CONFIG_KEY_OAUTH_TOKEN)
+
+		config.Set(configuration.AUTHENTICATION_TOKEN, "") // clear token to avoid using it during authentication
+
+		regions := config.GetStringSlice(configuration.SNYK_REGION_URLS)
```

Comment:
Suggestion: This logic is a duplication of the default function changes, let's try to apply DRY here please.